### ISSUE TITLE
cmake: missing header in v1.2.3

### ISSUE THREAD

**Issue by nim65s** [stack-of-tasks/pinocchio#380]
```
In file included from pinocchio-1.2.3/bindings/python/multibody/fcl/expose-fcl.cpp:19:0:
pinocchio-1.2.3/build/include/pinocchio/bindings/python/multibody/fcl/contact.hpp:21:59: erreur fatale : pinocchio/spatial/fcl-pinocchio-conversions.hpp : Aucun fichier ou dossier de ce type
 #include "pinocchio/spatial/fcl-pinocchio-conversions.hpp"
```
It should be added here: https://github.com/stack-of-tasks/pinocchio/blob/v1.2.3/CMakeLists.txt#L160

**Comment by jcarpent** [comment]
I do not have any trouble with the release 1.2.3. Can you tell more about your issue?

**Comment by nim65s** [comment]
I can't reproduce… Sorry about that.

**Comment by proyan** [comment]
Same problem in Kevin's system as well. There is no installation of HPP-FCL, but it is apparently not removed inside the bindings. I think the header should be conditional on the availability of Hpp-fcl. Right?

Current version: 1.2.3
System: 14.04, 64-bit

`In file included from /usr/HRP2/pinocchio/bindings/python/multibody/fcl/expose-fcl.cpp:19:0:
/usr/HRP2/pinocchio/_build_release/include/pinocchio/bindings/python/multibody/fcl/contact.hpp:21:59: fatal error: pinocchio/spatial/fcl-pinocchio-conversions.hpp: No such file or directory
 #include "pinocchio/spatial/fcl-pinocchio-conversions.hpp"`

**Comment by nim65s** [comment]
It's weird… But yes, the header is already conditionally available: 
https://github.com/stack-of-tasks/pinocchio/blob/v1.2.3/CMakeLists.txt#L272

I tried to recompile without HPP-FCL and it worked… I'll try in a clean container tonight.

**Comment by proyan** [comment]
> It's weird… But yes, the header is already conditionally available:
https://github.com/stack-of-tasks/pinocchio/blob/v1.2.3/CMakeLists.txt#L272

The problem is in python bindings `pinocchio/bindings/python/multibody/fcl/contact.hpp`
If I understand correctly, the files in the bindings/python/multibody/fcl folder are for exposing fcl objects, so they should not be included at all

**Comment by jcarpent** [comment]
Solved by #383.
